src/games/strategy/triplea/ai/fastAI/FastOddsEstimator.java

```diff
@@ -0,0 +1,146 @@
+package games.strategy.triplea.ai.fastAI;
```

Comment:
Actually while the clone data is slow, the majority of time is spend on casualty sorting in the regular battle calc. To remove the data clone, you'd essentially have to rewrite the entire battle calc since it actually plays out the battle each time which changes the game data. The goal of adding this is to create a simple formula based battle calc to avoid the data clone and actual battle calcs. This makes the AI 2x-10x faster though doesn't play as well.